### PR TITLE
Finish moving docs from ouroboros network

### PR DESCRIPTION
**Pull request by dnadales**
Part of https://github.com/input-output-hk/ouroboros-network/issues/4145, and required for https://github.com/input-output-hk/ouroboros-network/pull/4374.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,12 +132,28 @@ are always welcome.
 
 ## Formatting the code
 
-We use `stylish-haskell` for Haskell code formatting. There is a CI
-script (TODO: link to it) that checks that the code is properly
-formatted.
+We use `stylish-haskell` >= 0.11.0.0 for Haskell code formatting. There is a CI
```
Comment (jorisdral) [inline CONTRIBUTING.md:135]: I think we should update this to `>= 0.14.3.0`, which is the version that CI uses (IIRC)

File: docs/Introduction.md [hunk 2]
```diff
@@ -17,10 +17,13 @@ This document provides an index to different Consensus documents:
   foundations.
 - [Style guide](StyleGuide.md): describes the style guide we use when developing
   Consensus.
-- [Testing](Testing.md): describes our approach for testing consensus.
-- [Interface changelog](interface-CHANGELOG.md): lists changes that are
-  important for downstream Cardano development teams.
+- [Interface changelog](interface-CHANGELOG.md): contains the old changelog,
+  which describes previos changes that are important for downstream Cardano
```
Comment (jorisdral) [inline docs/Introduction.md:21]:
```suggestion
  which describes previous changes that are important for downstream Cardano
```

**Review by jorisdral — APPROVED** [review]
Another glorious diff :smile: 

image

**Review by abailly-iohk — APPROVED** [review]
LGTM. Perhaps would be useful to add more structure to group "all" genesis stuff in a single place but that can be done later.

File: CONTRIBUTING.md (hunk 1 above)
Comment (dnadales) [reply CONTRIBUTING.md:135]: Thanks, done!

**Pull request merged**